app/components/widgets/ProgressSteps.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react';
+import classNames from 'classnames';
+import SvgInline from 'react-svg-inline';
+import iconTickSVG from '../../assets/images/widget/tick.inline.svg';
+import iconCrossSVG from '../../assets/images/widget/cross.inline.svg';
+import styles from './ProgressSteps.scss';
+
+type Props = {
+  stepsList: Array<string>,
+  progressInfo: {
+    currentIndex : number,
+    error?: boolean
+  }
+};
+
+export default class ProgressSteps extends Component<Props> {
+
+  createSetps = () => {
+    const { stepsList, progressInfo } = this.props;
+    // progressIndex can't be less than 0
```

Comment:
if so maybe we don’t need this code.

Reply:
https://github.com/Emurgo/yoroi-frontend/commit/98709843722ed54bf5602827380963aa90773e9e